enterprise/server/backends/prom/prom.go

```diff
@@ -39,7 +39,7 @@ var (
 			ExportedFamily: &dto.MetricFamily{
 				Name: proto.String("exported_buildbuddy_remote_execution_queue_length"),
 				Help: proto.String("Number of actions currently waiting in the executor queue."),
-				Type: dto.MetricType_GAUGE.Enum(),
+				Type: dto.MetricType_COUNTER.Enum(),
```

Comment:
should this one be a gauge still? the queue length metric we have for the executors today is a gauge

Reply:
Done